### PR TITLE
Improve build workflow

### PR DESCRIPTION
**Pull request by zigarn**
https://github.com/Zenika/alpine-chrome/actions/runs/2494874105 vs https://github.com/Zenika/alpine-chrome/actions/runs/2490926055

### CODE REVIEW

File: with-puppeteer-xvfb/test.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+DIR=$(cd $(dirname $0) && pwd)
+${DIR}/../test.sh
+WORK_DIR=$(mktemp --directory)
+#trap "rm -rf ${WORK_DIR}" EXIT
```
Comment (hgwood) [inline with-puppeteer-xvfb/test.sh:6]: Could be removed maybe?

**Review by hgwood — APPROVED** [review]
Very nice, thanks :)

File: with-puppeteer-xvfb/test.sh (hunk 1 above)
Comment (zigarn) [reply with-puppeteer-xvfb/test.sh:6]: No, should be uncommented, forgot it thanks.

Comment (zigarn) [reply with-puppeteer-xvfb/test.sh]: Fixed.

**Pull request merged**